src/main/java/org/fenixedu/legalpt/services/raides/process/RaidesService.java

```diff
@@ -68,6 +64,12 @@ public class RaidesService {
     private static final int MAX_OTHER_SCHOOL_LEVEL_LENGTH = 80;
     private static final String TECHNICAL_SPECIALIZATION = "TECHNICAL_SPECIALIZATION";
     protected LegalReport report;
+    private static final CompetenceCourseType dissertationCompetenceCourseType =
```

Comment:
There is no open transaction when initialing these static fields, no you can't read domain objects